DEPS

```diff
@@ -52,12 +54,72 @@ else:
 print "Using make %s" % make
 
 here = os.getcwd()
-install = os.path.join(here, "install")
 
 hooks = [
     {
-        "name": "deps",
-        "pattern": ".",
-        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "INSTALL_DIR=%s"%install],
+        "name": "libunwind",
+        "pattern": "^libunwind/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_libunwind" ],
+    },
+    {
+        "name": "tcmalloc",
+        "pattern": "^tcmalloc/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_tcmalloc" ],
     },
+    {
+        "name": "openssl",
+        "pattern": "^openssl/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_openssl" ],
+    },
+    {
+        "name": "protobuf",
+        "pattern": "^protobuf/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_protobuf" ],
+    },
+    {
+        "name": "libevent",
+        "pattern": "^libevent/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_libevent" ],
+    },
+    {
+        "name": "libevhtp",
+        "pattern": "^libevhtp/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_libevhtp" ],
+    },
+    {
+        "name": "gflags",
+        "pattern": "^gflags/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_gflags" ],
+    },
+    {
+        "name": "glog",
+        "pattern": "^glog/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_glog" ],
+    },
+    {
+        "name": "ldns",
+        "pattern": "^ldns/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_ldns" ],
+    },
+    {
+        "name": "sqlite3",
+        "pattern": "^sqlite3/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_sqlite3" ],
+    },
+    {
+        "name": "leveldb",
+        "pattern": "^leveldb/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_leveldb" ],
+    },
+    {
+        "name": "json-c",
+        "pattern": "^json-c/",
+        "action": [ make, "-f", os.path.join(here, "certificate-transparency/build.gclient"), "_json-c" ],
+    },
+    # Do this last
```

Comment:
Yeah, that's sad making, and is why I had originally moved all of this into the one `gclient.build` Makefile so that  I could outline the deps.  But, as I told you about offline, that brings its own set of issues (mainly around compile time on Travis).

Probably more can be done in this area later on, though.